src/components/activities/Activity.jsx

```diff
@@ -24,15 +24,20 @@ const renderInfo = (poster, props) => {
   )
 }
 
+const countDecimals = number => {
```

Comment:
It seems a bit counterintuitive to me to implement a whole new function for such a 'basic' check? But for now, it'll do...